lib/rubocop/formatter/tap_formatter.rb

```diff
@@ -0,0 +1,70 @@
+# frozen_string_literal: true
+
+module RuboCop
+  module Formatter
+    # This formatter formats report data in the TAP format.
```

Comment:
I'd extend the description here a bit for the sake of people like me, who had never heard of this before. :-)